cfgov/v1/management/commands/update_data_snapshot_values.py

```diff
@@ -0,0 +1,79 @@
+import json
+import logging
+import os
+
+from django.core.management.base import BaseCommand
+
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Monthly updates to data snapshot values'
+
+    def expand_path(self, path):
+        """Expands a relative path into an absolute path"""
+        rootpath = os.path.abspath(os.path.expanduser(path))
+
+        return rootpath
+
+    def add_arguments(self, parser):
+        """Adds all arguments to be processed."""
+        parser.add_argument(
+            '--snapshot_file',
+            required=True,
+            help='JSON file containing all markets\' data snapshot values'
+        )
+
+    def get_data_snapshots(self):
+        """ Gets all data snapshots from browse pages
+        Assumes there is one data snapshot per page
+        """
+        snapshots = []
+        for page in BrowsePage.objects.all():
+            stream_data = page.specific.content.stream_data
+            snapshot = filter(
+                lambda item: item['type'] == 'data_snapshot',
+                stream_data
+            )
+            if snapshot:
+                snapshot[0]['value']['page'] = page
+                snapshots.append(snapshot[0]['value'])
+        return snapshots
+
+    def find_data_snapshot(self, market_key, snapshots):
+        """ Look up data snapshot by the provided market key
+        Assumes there is one data snapshot per key
+        """
+        snapshot = filter(
+            lambda item: item['market_key'] == market_key,
+            snapshots
+        )
+        if snapshot:
+            return snapshot[0]
+        return None
```

Comment:
One additional thing: I'm not sure this code would work properly in Python 3, where `filter` returns a generator instead of a list. This would also be an issue on [this line](https://github.com/cfpb/cfgov-refresh/pull/3091/files/e1d5f62d7f2d6c5b0f7f7ff42962b47277598e73#diff-54080eea21e188ef2f4b2d1969ad2233R37) where you check the result of a `filter` in an `if` statement. I know we're not on Python 3 yet but it might be good to get into the habit for the future.

Here's an example difference in behavior. In Python 2 you can evaluate the result of the `filter` as a boolean:

```sh
$ python
Python 2.7.11 (default, Mar  7 2016, 09:55:43) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.72)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> x = [1, 2, 3]
>>> y = filter(lambda z: z > 4, x)
>>> bool(y)
False
>>> type(y)
<type 'list'>
```

but in Python 3 this doesn't work properly because it returns an object of type `filter`.

```
$ python3
Python 3.5.1 (default, Feb 12 2016, 16:08:42) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.72)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> x = [1, 2, 3]
>>> y = filter(lambda z: z > 4, x)
>>> bool(y)
True
>>> type(y)
<class 'filter'>